core/src/main/java/org/infinispan/configuration/parsing/Parser80.java

```diff
@@ -2311,35 +2311,61 @@ private void parseIndexing(final XMLExtendedStreamReader reader, final Configura
                throw ParseUtils.unexpectedAttribute(reader, i);
          }
       }
-      Properties indexingProperties = parseProperties(reader);
+
+      Properties indexingProperties = new Properties();
+      while (reader.hasNext() && (reader.nextTag() != XMLStreamConstants.END_ELEMENT)) {
+         Element element = Element.forName(reader.getLocalName());
+         switch (element) {
+            case INDEXED_ENTITY: {
```

Comment:
Oh, I did not realise you may be actually asking for a parent ```<indexed-entities>``` element to hold them all. Is that what you intended?